### PR TITLE
FEATURES# add models and migration files

### PR DESCRIPTION
**Pull request by Orlayhemmy**
#WHAT DOES THIS PR DO?
It adds sequelize file

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +31,9 @@
     "istanbul": "^0.4.5",
     "istanbul-coveralls": "^1.0.3",
     "nyc": "^11.3.0",
+    "pg": "^7.4.0",
+    "pg-hstore": "^2.3.2",
+    "sequelize": "^4.23.1",
     "validator": "^9.1.1"
   },
   "engines": {
```
Comment (ifiokeyo) [inline package.json:39]: why do u have this option `engines` ?

File: server/app.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const app = express();
 
 
 app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.urlencoded({extended: true}));
```
Comment (ifiokeyo) [inline server/app.js:9]: why did you change the extended `options` to `true` ?

File: server/config/config.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "development": {
+    "username": "postgres",
+    "password": "profyem001",
+    "database": "CenterManager",
```
Comment (ifiokeyo) [inline server/config/config.json:5]: You did not specify the `port` for the db type. Why is that so?

File: server/migrations/20171128031456-create-user.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      email: {
+        type: Sequelize.STRING,
+        unique: true
+      },
+      password: {
+        type: Sequelize.STRING
+      },
+      role: {
+        type: Sequelize.STRING,
+        defaultValue: 0
```
Comment (ifiokeyo) [inline server/migrations/20171128031456-create-user.js:23]: Why do have a default value for the `role` field ?

File: server/models/center.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+      type: DataTypes.STRING,
+      allowNull: false
+  },
+  facilities: {
+    type: DataTypes.STRING,
+    unique: true,
+    allowNull: false
+},
+  }, {
+    classMethods: {
```
Comment (ifiokeyo) [inline server/models/center.js:23]: I don't think class methods work for `pg version` 7 upwards. Read the docs for clarity.

File: package.json (hunk 1 above)
Comment (Orlayhemmy) [reply package.json:39]: that was a mistake when i was trying to make code climate work and I forgot to remove it. I will rectify

File: server/config/config.json (hunk 3 above)
Comment (Orlayhemmy) [reply server/config/config.json:5]: I forgot. I will rectify

File: server/app.js (hunk 2 above)
Comment (Orlayhemmy) [reply server/app.js:9]: Since false only accepts values of a key pair to be string or array, I just thought I should change it to true so that it can accept any type, just in case

File: server/migrations/20171128031456-create-user.js (hunk 4 above)
Comment (Orlayhemmy) [reply server/migrations/20171128031456-create-user.js:23]: I have changed it already but I thought of using 0 and 1 for differentiating between roles, thats why I put it there

File: server/models/center.js (hunk 5 above)
Comment (Orlayhemmy) [reply server/models/center.js:23]: ok. thanks
